Twilio Unified Login [DOC-757] (#5756)
segmentio/segment-docs: src/segment-app/iam/secure-password.md

Picking a strong password is one of the most important things you can do to protect your account.

> info "Twilio Unified Login users can manage their password in their Twilio account"

> > Twilio Unified Login users can manage their user settings, including name, email, password and 2FA settings, directly in their Twilio account. To learn more about Twilio’s user and password policies, review Twilio's [Account Management](https://support.twilio.com/hc/en-us/sections/205104908-Account-Management){:target="_blank”} documentation. 

Reviewer: ```suggestion
> Twilio Unified Login users can manage their user settings, including name, email, password, and 2FA settings, directly in their Twilio account. To learn more about Twilio’s user and password policies, review Twilio's [Account Management](https://support.twilio.com/hc/en-us/sections/205104908-Account-Management){:target="_blank”} documentation. 
```

### Under the Hood

> When you first create a Segment account, or when you reset or change the password of an existing account, you'll see some tools which Segment uses to help you choose a strong password. We use [zxcvbn](https://blogs.dropbox.com/tech/2012/04/zxcvbn-realistic-password-strength-estimation/) to show your password strength, and [Have I Been Pwned](https://haveibeenpwned.com/Passwords) to notify you if your password has been found in any data breaches. Your password is never stored in plaintext, and is securely stored using the [bcrypt](https://en.wikipedia.org/wiki/Bcrypt) password hashing function in our database.

Reviewer: ```suggestion
When you first create a Segment account, or when you reset or change the password of an existing account, you'll see some tools which Segment uses to help you choose a strong password. Segment uses [zxcvbn](https://blogs.dropbox.com/tech/2012/04/zxcvbn-realistic-password-strength-estimation/){:target="_blank”} to show your password strength, and [Have I Been Pwned](https://haveibeenpwned.com/Passwords){:target="_blank”} to notify you if your password has been found in any data breaches. Your password is never stored in plaintext, and is securely stored using the [bcrypt](https://en.wikipedia.org/wiki/Bcrypt){:target="_blank”} password hashing function in Segment's database.
```
